### PR TITLE
modifying resource allocation for alertmanager as it should have been for Prometheus instead

### PR DESCRIPTION
**Pull request by geekbass**
**What type of PR is this?** Bug

**What this PR does/ why we need it**: Causes failure for clusters to launch Prometheus addon and the resources that were set for alertmanager were supposed to be set for Prometheus instead based on soak findings.

The original PR was not set properly.

**Which issue(s) this PR fixes**:

`[tiller] 2020/07/02 15:37:42 warning: Rollback "prometheus-kubeaddons" failed: failed to create resource: Alertmanager.monitoring.coreos.com "prometheus-kubeaddons-prom-alertmanager" is invalid: [spec.resources.limits.cpu: Invalid value: "integer": spec.resources.limits.cpu in body must be of type string: "integer", spec.resources.requests.cpu: Invalid value: "integer": spec.resources.requests.cpu in body must be of type string: "integer"]`

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Checklist**

* [x] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

**Comment by sebbrandt87** [comment]
this cries for a test too 😬 but not now
just fix it like this 👍

**Comment by gracedo** [comment]
I think we need to make a new revision `-10` with these config changes

File: addons/prometheus/0.38.x/prometheus-10.yaml [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+metadata:
+  name: prometheus
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: prometheus
+    # TODO: we're temporarily supporting dependency on an existing default storage class
+    # on the cluster, this hack will trigger re-queue on Addons until one exists.
+    kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.38.1-9"
```
Comment (gracedo) [inline addons/prometheus/0.38.x/prometheus-10.yaml:13]: this need to be updated to `-10`!

**Comment by gracedo** [comment]
sorry everything is so manual, but to make it easier for reviewers, it's recommended:
`To make it easier for reviewers to review PRs where revisions are [manually] added make the first commit in your branch a commit to ONLY copy the previous revision to the new revision file. That way the following commits can actually include your changes and will be easier to historically follow without needing to manually diff the files.`

**Pull request closed without merging**